CHANGELOG.md

```diff
@@ -1,6 +1,7 @@
 # 3.6.2 (2020-01-23)
 
 - Fixed the character count directive to not affect the underlying control value. [#95](https://github.com/blackbaud/skyux-forms/pull/95)
+- Fixed the single file attachment component to use resource strings. [#93](https://github.com/blackbaud/skyux-forms/pull/93)
```

Comment:
"Fixed ... to use" reads a little oddly to me, but I don't think it's a big deal for a changelog entry. I think the entry would read more naturally if we changed "Fix" to "Updated" or as an alternative if we changed "to use" to "so that it can use." But I went ahead and approved as-is because I don't think either change is strictly necessary. ... If you decide to change the entry, give me a heads up, and I'll re-approve. Thanks!

Reply:
Changed it to "Updated". Thanks for the quick review @blackbaud-johnly !